scripts/generate-values/base-renku-values.yaml.template

```diff
@@ -88,3 +88,16 @@ postgresql:
   postgresqlPassword: <use `openssl rand -hex 32`>
 redis:
   password: <use `openssl rand -hex 32`>
+ui:
+  baseUrl: https://{renku_domain}
+  gatewayUrl: https://{renku_domain}/api
+  client:
+    uiserverUrl: https://{renku_domain}/ui-server
```

Comment:
I don't see this set in the default values in the renku chart - why is it set here? Seems like this default should either be set on the component chart or at least on the renku chart.

Reply:
But more importantly, can't this always be built from the information contained in the `ui.server.serverData` values? Since the two charts are now unified, you shouldn't need to declare this manually.

Reply:
Do you mean we should add it (commented) to `helm-chart/renku/values.yaml` ? Or we should drop it since it's set as a default in the UI values? If it's the latter, then we should probably drop  `baseUrl` and `gatewayUrl` too since the default is already derived from `ui.protocol` and `.Values.global.renku.domain`.

Reply:
> can't this always be built from the information contained in the ui.server.serverData values?
Yes, we can unify some of the values from `client` and `server` but that requires a bit more work since the URLs are built in different ways and it would make sense to unify the logic -- should I make an issue in the UI to take care of that?

Reply:
yes please do, there is certainly an opportunity here to reduce redundancy

Reply:
drop as much as possible from this template (but make sure it still yields a usable values file) because it's meant as a first exposure for someone deploying renku

Reply:
This is the issue in the UI https://github.com/SwissDataScienceCenter/renku-ui/issues/1976

I removed all the values except the `welcomePage` since they are not strictly needed.